tests/plots/test_mapping.py

```diff
@@ -66,6 +66,33 @@ def test_globe_spheroid():
     assert globe_params['a'] == 6367000
     assert globe_params['b'] == 6360000
 
+def test_aea():
```

Comment:
Can you add a blank line before this to satisfy the robots?

Reply:
It doesn't look like you actually added the line here...

Reply:
sigh.  okay, *should* be fixed now.